README.md

```diff
@@ -169,13 +171,13 @@ If a pull requests is found with the label `major`/`minor`/`patch`, the correspo
 
 You can use any of the following variables in `change-template`:
 
-| Variable  | Description                                                                 |
-| --------- | --------------------------------------------------------------------------- |
-| `$NUMBER` | The number of the pull request, e.g. `42`.                                  |
-| `$TITLE`  | The title of the pull request, e.g. `Add alien technology`.                 |
-| `$AUTHOR` | The pull request author’s username, e.g. `gracehopper`.                     |
-| `$BODY`   | The body of the pull request e.g. `Fixed spelling mistake`.                 |
-| `$URL`    | The URL of the pull request e.g. `https://github.com/octocat/repo/pull/42`. |
+| Variable  | Description                                                                                                                                                                                                                                 |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `$NUMBER` | The number of the pull request, e.g. `42`.                                                                                                                                                                                                  |
+| `$TITLE`  | The title of the pull request, e.g. `Add alien technology`. Any characters matching `change-title-escapes` will be prepended with a backslash so that they will appear verbatim instead of being interpreted as markdown format characters. |
```

Comment:
What about escaping `@`
There are two options according to https://github.com/github/markup/issues/1168

One using zero-width space `​` or html comment `<!-- -->`. I think the html comment is better as it is easier to "recreate" not everybody knows how to type a zero-width space.

@​jetersen
@<!-- -->jetersen

Reply:
I like the zero-width space better as it doesn't clutter reading as much. We are auto-generating these lines anyways.

Reply:
What if user accidentally deletes the line and does not know how to type a zero-width space?

Reply:
@jetersen @&nbsp;jetersen

Reply:
Oops it's a nbsp

Reply:
@<!---->jetersen

Reply:
Maybe using comments is the way to go then.

Reply:
You can also do this:
`@&ZeroWidthSpace;jetersen` @&ZeroWidthSpace;jetersen
`@&nbsp;jetersen` @&nbsp;jetersen

`&nbsp;` is not a good option

Reply:
`<!---->` @<!---->​jetersen
`&#x2060;` @&#x2060;jetersen
`&#xFEFF;` @&#xFEFF;jetersen
Comments seem to be the most obvious and short option.

Reply:
Same trick can be used for #s.